packages/app/src/components/Bookmarks/BookmarkFolderItem.tsx

```diff
@@ -162,13 +167,87 @@ const BookmarkFolderItem: FC<BookmarkFolderItemProps> = (props: BookmarkFolderIt
     mutateBookmarkInfo();
   }, [mutateBookmarkInfo, mutateParentBookmarkFolder]);
 
+  const [, bookmarkFolderDragRef] = useDrag({
+    type: 'FOLDER',
+    item: props,
+    end: (item, monitor) => {
+      const dropResult = monitor.getDropResult();
+      if (dropResult != null) {
+        mutateParentBookmarkFolder();
+      }
+    },
+    collect: monitor => ({
+      isDragging: monitor.isDragging(),
+      canDrag: monitor.canDrag(),
+    }),
+  });
+
+
+  const folderItemDropHandler = async(item: BookmarkFolderItemProps) => {
+    try {
+      await apiv3Put('/bookmark-folder', { bookmarkFolderId: item.bookmarkFolder._id, name: item.bookmarkFolder.name, parent: bookmarkFolder._id });
+      await mutateChildBookmarkData();
+      toastSuccess(t('toaster.update_successed', { target: t('bookmark_folder.bookmark_folder') }));
```

Comment:
## Check Request
i18n is not working if use `bookmark_folder.bookmark_folder`, please check grep: `bookmark_folder`.
Now `bookmark_folder` json is in the translation.json. But some toastSuccess use `ns: 'commons'`.
ex. `toastSuccess(t('toaster.update_successed', { target: t('bookmark_folder.bookmark_folder'), ns: 'commons' }));`

Reply:
@mudana-grune 
It is not working. Please check be careful.
![image](https://user-images.githubusercontent.com/68407388/216928135-0d4eb8cf-0555-4a09-b649-fdc536f0c550.png)